### PR TITLE
Simplify caldavtester setup

### PR DESCRIPTION
**Pull request by untitaker**
See #29

This runs well, but I don't understand why all tests are ignored.

### CODE REVIEW

File: compat/xandikos-caldavtester.sh [hunk 1]
```diff
@@ -8,10 +8,5 @@ CFGDIR=$(readlink -f $(dirname $0))
 
 run_xandikos --defaults
 
-if which testcaldav >/dev/null; then
```
Comment (jelmer) [inline compat/xandikos-caldavtester.sh]: Please leave this in place, it allows me to use the system caldav-tester.

File: compat/testcaldav.sh [hunk 2]
```diff
@@ -1,8 +1,9 @@
-#!/bin/bash -e
+#!/bin/bash
+set -e
```
Comment (jelmer) [inline compat/testcaldav.sh]: Why the use of set rather than hashbang?

**Comment by jelmer** [comment]
To run the tests, you need to enable one of the <feature> lines in compat/serverinfo.xml. None of the tests currently pass though.

**Comment by jelmer** [comment]
one of the &lt;feature&gt; lines. Hopefully github won't strip the tag out this time.

**Comment by untitaker** [comment]
Fixed according to your comments.

**Comment by jelmer** [comment]
Merged, thanks!

**Pull request closed without merging**